api/trello/api.go

```diff
@@ -0,0 +1,83 @@
+package trello
+
+import(
+  "net/url"
+  "net/http"
+  "encoding/json"
+  "io/ioutil"
+)
+
+const API_KEY = "[API_KEY]"
+
+var LinkForAuthKey = "https://trello.com/1/authorize?expiration=never&scope=read,write&response_type=token&name=Treit&key=" + API_KEY
+
+func CreateNewBoard(authKey string, boardname string) (board Board) {
```

Comment:
Also return the error here. The user of the package should handle the error.

Reply:
I would return a pointer to Board, so that you can return nil if an error occursed.